flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/SchedulingStrategy.java

```diff
@@ -40,23 +40,23 @@
 	/**
 	 * Called whenever vertices need to be restarted (due to task failure).
 	 *
-	 * @param verticesNeedingRestart The tasks need to be restarted.
+	 * @param verticesNeedingRestart The tasks need to be restarted
 	 */
 	void restartTasks(Set<ExecutionVertexID> verticesNeedingRestart);
```

Comment:
Name of parameter should be `verticesToRestart`

Reply:
Done